pkg/util/configspec.go

```diff
@@ -80,6 +81,38 @@ func UnmarshalConfigSpecFromBase64XML(
 	return UnmarshalConfigSpecFromXML(data)
 }
 
+// MarshalConfigSpecToJSON returns a byte slice of the provided ConfigSpec
+// marshalled to a JSON string.
```

Comment:
```suggestion
// marshaled to a JSON string.
```
marshalled -> marshaled.

Reply:
Hmm, I think it works either way. I've never been sure, and I know I've used it both ways. Since you picked the nit, I decided [to check](https://sapling.ai/usage/marshaled-vs-marshalled). It is not even close 😃

Reply:
😆  I dropped this because I am pretty sure the linter has complained about incorrect spelling for this in my changes.  Also, the golang JSON package also spells it as marshaled/unmarshaled (https://pkg.go.dev/encoding/json, search for marshaled/unmarshaled).  But anyway, I will leave it to you if you feel strongly about this.

Reply:
Ah, good catch about what Golang uses. You'd think if know given I was just in that code :)

Reply:
I actually used `Marshaled` on the field definition :)